styles.css

```diff
@@ -507,3 +507,185 @@ article {
   opacity: 0.9;
   cursor: pointer;
 }
+
+/*
+/* ----------------------form area-------- */
+#form-area1 {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+#text-box {
+  width: 80%;
+}
+
+.form-txt {
+  width: 327px;
+  height: 216px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  margin: 25px;
+  color: #172b4d;
+}
+
+#message {
+  position: relative;
+  left: 12px;
+  right: 12px;
+  top: 12px;
+  bottom: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+}
+
+input {
+  width: 280px;
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-color: #dbd8d7;
+  padding: 0.6em;
+  margin: 15px 20px;
+  color: #dbd8d7;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+}
+
+.placeholder {
+  height: 50px;
+}
+
+textarea {
+  background: #fbf8f7;
+  position: absolute;
+  align-items: center;
+  padding-left: 12px;
+  font-style: italic;
+  resize: both;
+  border: none;
+  outline: none;
+  height: 90px;
+  width: 303px;
+  left: 12px;
+  top: 12px;
+}
+
+input[type="textarea"]::placeholder {
+  color: #979493;
+  font-style: italic;
+}
```

Comment:
- [ ] Great work attempting to implement the change suggested by the first reviewer. The way you are selecting this element is wrong and that is the reason why the style declarations are not reflected in the UI. The correct way to select this element is `textarea::placeholder` because there is no `input` tag with `type="textarea"` in HTML. 👍 😎

Reply:
hi [Gambit142](https://github.com/Gambit142)
it resolved now
only textarea up dated for this